### ISSUE TITLE
unsuscribe from the initialize?

### ISSUE THREAD

**Issue by robertppn** [pie6k/jquery.initialize#24]
Hi,
Is there a way to unsubscribe from the initialize? It keeps firing when the target control is removed and added back to the DOM.
thx!

**Comment by dbezborodovrp** [comment]
This is not currently supported.

**Comment by dbezborodovrp** [comment]
I tentatively propose an API as follows:

```
var obs = $.initialize([selector], [callback]);
obs.disconnect();
```

This would be consistent with `MutationObserver.disconnect()`.

**Comment by dbezborodovrp** [comment]
To achieve this, we could simply return the `MutationObserver` directly from `$.initialize()`, although this could make it difficult if we ever want to add more functionality to the object in the future.

**Comment by dbezborodovrp** [comment]
This has been implemented in 1.3.0. See 7f20aa11ddafd3c72e4f1a51c06057983d80d187.
